src/test/functional/tests/loginPage.spec.ts

```diff
@@ -10,13 +10,13 @@ test.describe("Login Page", () => {
     await loginPage.initialize();
   });
 
-  test("Test Case 1: Login As An Admin", async ({ page }) => {
+  test("Test 1: Admin Should Be Logged In", async ({ page }) => {
     await loginPage.navigateToMainPage()
     await loginPage.loginUser(adminUserTestData.userName, adminUserTestData.password)
     await expect(loginPage.userNameAfterLogin).toHaveText(loginPageAssertion.adminName);
   });
 
-  test("Test Case 2: Login As A User", async ({ page }) => {
+  test("Test 2: User Should Be Logged In", async ({ page }) => {
```

Comment:
nitpick: Admin is user as well - so perhaps "Test 2: Regular User Should Be Logged In"? And why do you use first uppercase character in every single word in the test name? - I'm just curious.

Reply:
I like to use capital letter